### ISSUE TITLE
MIC doesn't work on Lovelace cards

### ISSUE THREAD

**Issue by JoshuaSeidel** [home-assistant/iOS#2958]
**iOS device model, version and app version**

Model Name: iPhone 15 pro
Software Version: ios18.1
App version: 2024.9 (2024.803)

**Home Assistant Core Version**
8.2024

**Describe the bug**
When using a Lovelace card like frigate, and you have two way audio, the Lovelace card does not have access to the phone's microphone. Assist works fine with the mic. But not any Lovelace cards
**To Reproduce**
Have a Lovelace frigate card attached to a Reolink Poe doorbell and try to utilize 2way via the card

**Expected behavior**
Pushing and holding the mic button on the card should engage the mic on the device like it does in safari or chrome and send auto back through the frigate settings like the card is designed to do

**Comment by bgoncal** [comment]
Are you accessing your HA using HTTP or HTTPS?

**Comment by JoshuaSeidel** [comment]
> Are you accessing your HA using HTTP or HTTPS?

Https

**Comment by bgoncal** [comment]
Does it shows any error?

**Comment by JoshuaSeidel** [comment]
No error just doesn't engage. I am using the beta can see if TestFlight lets me send logs

**Comment by JoshuaSeidel** [comment]
![image](https://github.com/user-attachments/assets/5f961f51-23a4-46c3-a5e6-dec59f40e80c)

**Comment by bgoncal** [comment]
Does it work in iOS Safari? (Not macOS)
Have you checked with the component creator if this is a known issue?

**Comment by JoshuaSeidel** [comment]
Yes I have tried with safari and chrome on my iPhone and works fine. Yes I've checked with the card creator. If it works fine on my iPhone in safari and chrome. Then the only variable is the app. That's the only place I experience the problem.

**Comment by bgoncal** [comment]
My question was more in the sense that the custom card creator may already know the reason. I will see if I can make a setup environment this week with my unifi cameras and reproduce the issue.
Can you share you configuration so I reproduce the same?
